spec/features/trainees/degrees/editing_degree_spec.rb

```diff
@@ -70,9 +70,8 @@ def when_i_visit_the_edit_degree_details_page
                                   id: trainee.degrees.first.id)
   end
 
-  def and_i_am_redirected_to_the_summary_page
-    summary_page.load(id: trainee.id)
-    expect(summary_page).to be_displayed(id: trainee.id)
+  def then_i_am_redirected_to_confirm_page
+    expect(page.current_path).to eq("/trainees/#{trainee.id}/degrees/confirm")
```

Comment:
Can we add a page object? We're using them for everything else so should probably be consistent here.

Reply:
Added a token page object for consistence.